examples/tutorials/analysis-2d/ring_background.py

```diff
@@ -215,20 +216,21 @@
 print(stacked_on_off)
 
 
-######################################################################
+# #####################################################################
 # Compute correlated significance and correlated excess maps
 # ----------------------------------------------------------
 #
 # We need to convolve our maps with an appropriate smoothing kernel. The
 # significance is computed according to the Li & Ma expression for ON and
 # OFF Poisson measurements, see
 # `here <https://ui.adsabs.harvard.edu/abs/1983ApJ...272..317L/abstract>`__.
-# Since astropy convolution kernels only accept integers, we first convert
-# our required size in degrees to int depending on our pixel size.
 #
+#
+# Also, since the off counts are obtained with a ring background estimation, then we specify `correlate_off=False`
```

Comment:
```suggestion
# Also, since the off counts are obtained with a ring background estimation, and are thus already correlated, we specify `correlate_off=False`
```